fe/fe-core/src/main/java/com/starrocks/connector/jdbc/JDBCMetadata.java

```diff
@@ -160,12 +164,22 @@ public List<Column> listPartitionColumns(String databaseName, String tableName,
     public List<PartitionInfo> getPartitions(Table table, List<String> partitionNames) {
         try (Connection connection = getConnection()) {
             List<Partition> partitions = schemaResolver.getPartitions(connection, table);
+            String minInt = IntLiteral.createMaxValue(Type.INT).getStringValue();
+            String minDate = DateLiteral.createMaxValue(Type.DATE).getStringValue();
```

Comment:
```suggestion
            String maxDate = DateLiteral.createMaxValue(Type.DATE).getStringValue();
```
minInt -> maxInt?
minDate -> maxDate?